packages/redux/src/bags/actions/setBagPromocodes.ts

```diff
@@ -0,0 +1,8 @@
+import { putBagPromocodes } from '@farfetch/blackout-client';
+import { setBagPromocodesFactory } from './factories';
+
+/**
+ * Updates a specific bag item with the given params.
```

Comment:
```suggestion
 * Sets the passed promocodes to the bag.
```

Reply:
I will change that to be similar to the API definition:
`Sets the list of promocodes to be applied to the bag.`

Reply:
Changed 👍

Reply:
Much better than my suggestion, thanks!